example/application/container.tsx

```diff
@@ -75,6 +75,9 @@ const Viewer: React.FunctionComponent<ViewerProps> = ({ document }): React.React
       setTemplates(action.payload);
       setSelectedTemplate(action.payload[0].id);
     }
+    if (action.type === "TIMEOUT") {
```

Comment:
I just realised that maybe we have function to check the type ? Doesn't typesfe has a helper for this ?